service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderIssuedEventHandlerTest.java

```diff
@@ -41,12 +45,13 @@
 import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.getCMOIssuedCaseLinkNotificationParameters;
 import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.callbackRequest;
 import static uk.gov.hmcts.reform.fpl.utils.OrderIssuedNotificationTestHelper.getExpectedCaseUrlParameters;
+import static uk.gov.hmcts.reform.fpl.utils.TestDataHelper.DOCUMENT_CONTENT;
 import static uk.gov.hmcts.reform.fpl.utils.matchers.JsonMatcher.eqJson;
 
 @ExtendWith(SpringExtension.class)
```

Comment:
Do all tests need to be MockitoExtension instead? @lewisbirks

Reply:
In this situation it is fine to have `SpringExtension.class` as we are making use of spring functionality in these tests. If we were only using mockito functionality (mocks/spies, and injectMocks) then we could change it to `MockitoExtension.class`

Reply:
Cool. wanted to confirm thanks Lewis :)